### PR TITLE
Discussion: NATIVE-305: Prepare for testbirds

### PR DESCRIPTION
**Pull request by maxammann**
This PR is more like a discussion about the test usecases. Please add usecases/add notes to them and ask questions.

### CODE REVIEW

File: docs/test-surveys/13-survey-react-native-release.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+### UC1: Open the app and choose a city of your choice as a location and the language you are most comfortable with. Explore the app and familiarize yourself with it. Turn off your internet connection and explore the same city and a further on.
```
Comment (klinzo) [inline docs/test-surveys/13-survey-react-native-release.md]: What do you mean by `a further on`?

File: docs/test-surveys/13-survey-react-native-release.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+#### Describe the steps you took.
+
+*Input box*
+
+### UC2: Choose city "Stadt Regensburg" as your location, if you haven’t done so already. Find out when shops are usually open in Regensburg. Switch the language as soon as you found it.
```
Comment (klinzo) [inline docs/test-surveys/13-survey-react-native-release.md]: You should probably mention the goal should be to find the information without using the search.

File: docs/test-surveys/13-survey-react-native-release.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+### UC3: Find the same information as in UC2 by using the search icon in the navigation bar. 
+
+The search terms you need are "Einkaufen" (German), "Shopping" (German), "التسوّق" (Arabic/العربية)
+
+#### Did you find the information?
+
+* [ ] Yes
+* [ ] No
+
+### UC4: Choose city "Stadt Regensburg" as your location, if you haven’t done so already. Try to find a job in the "Sprungbrett into work" offer. Also try the "Lehrstellenradar".
```
Comment (klinzo) [inline docs/test-surveys/13-survey-react-native-release.md]: Why not use a different city here (or above)?

File: docs/test-surveys/13-survey-react-native-release.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+Note: You need a stable internet connection.
+
+#### Did you a job offer near Regensburg?
```
Comment (klinzo) [inline docs/test-surveys/13-survey-react-native-release.md]:
```suggestion
#### Did you find a job offer near Regensburg?
```

File: docs/test-surveys/13-survey-react-native-release.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+* [ ] Yes
+* [ ] No
+
+### UC5: Turn off your internet connection (WiFi + Data) and repeat usecases UC2-4.
+
+Note: If the app does show you what you require internet then it is not a bug!
```
Comment (klinzo) [inline docs/test-surveys/13-survey-react-native-release.md]:
```suggestion
Note: If the app does show you that an internet connection is required then it is not a bug!
```
Also this is not entirely true, is it? Already opened cities/languages should work.

File: docs/test-surveys/13-survey-react-native-release.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+* [ ] (Very difficult)
+
+### Q2: How easy did you think it was to find what you were looking for? (1: Very easy/6: Very difficult)
+
+* [ ] (Very Easy)
+* [ ] \-
+* [ ] \-
+* [ ] \-
+* [ ] \-
+* [ ] (Very difficult)
```
Comment (klinzo) [inline docs/test-surveys/13-survey-react-native-release.md]: Maybe add a question for general observations.

File: docs/test-surveys/13-survey-react-native-release.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+### UC6: Events
+
+// TODO
+
+### UC7: Sharing 
+
+// TODO
+
+### UC8: Feedback
```
Comment (klinzo) [inline docs/test-surveys/13-survey-react-native-release.md]: Maybe a separate usecase testing the language change would make sense.

**Review by steffenkleinle — COMMENTED** [review]
I just had a look at the react-native-survey. If more review is required, say so.

File: docs/test-surveys/10-survey-refguide.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+### UC7: Go to settings and change the Font Size. Try all the different Font Sizes and verify that the layout changes accordingly. 
+
+*Input box*
+
+### UC8: You want to learn German. Use the app to find out more information about how to learn German. Location does not matter. Describe your steps. What information did you find? 
+
+*Input box*
+
+###  UC9: Explore the app by yourself, the more bugs you find the higher your payout will be! Check different tabs and different settings, or try some of the following actions:
```
Comment (Schedulaar) [inline docs/test-surveys/10-survey-refguide.md]: What are tabs?

File: docs/test-surveys/13-survey-react-native-release.md (hunk 1 above)
Comment (Schedulaar) [inline docs/test-surveys/13-survey-react-native-release.md]:
```suggestion
### UC1: Open the app and choose a city of your choice as a location and the language you are most comfortable with. Explore the app and familiarize yourself with it. Turn off your internet connection and explore the same city further on.
```

File: docs/test-surveys/13-survey-react-native-release.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+### UC7: Sharing 
+
+// TODO
+
+### UC8: Feedback 
+
+// TODO
+
+### UC-FINAL: Explore the app by yourself, the more bugs you find the higher your payout will be! Check different tabs and different settings, or try some of the following actions:
```
Comment (Schedulaar) [inline docs/test-surveys/13-survey-react-native-release.md]: What are tabs?

File: docs/test-surveys/13-survey-react-native-release.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+
+- Minimizing the app and using others (multitasking)
+- Change font size in the settings of your smartphone
```
Comment (Schedulaar) [inline docs/test-surveys/13-survey-react-native-release.md]:
```suggestion
- Change font size in the system settings of your smartphone
```

**Review by michael-markl — APPROVED** [review]
I'm fine with the ones defined so far.
Not sure whether that makes sense, but should we ask to test for compatibility on older OS versions?

**Pull request closed without merging**